content/en/docs/architecture/release-gating.md

```diff
@@ -67,88 +36,215 @@ Good candidates include:
+one -- with an automatic fallback to the best a test ever was. For tests without a 
+presence in the basis (i.e., new tests), we enforce that new tests work at least 95% of the time.
+
+The goal is simple: ensure each release is as good as or better than the last.
+
+Every test in OpenShift is assigned to a Jira component (e.g. Installer,
+kube-apiserver, etc) and every job in OpenShift is assigned variants (Platform, Network,
+Topology, etc).  The grid is organized by components on the y-axis, and variants on the
+x-axis. Each grid coordinate then either has a red or green square; where red
+indicates that at least one test for that component isn't meeting the requirements for
+the dashboard.
+
+There are multiple dashboards available in Component Readiness, and new ones can be
+created for particular needs. There is always a "X.Y-main" dashboard that is used to
+determine a release's readiness to ship.
+
+{{< alert title="Note" color="info" >}}
+Regressions detected by component readiness are not candidates for automatic reverts 
+by TRT, but may be reverted by the team that made the change.
+{{< /alert >}}
+
+## Selecting a path
 
-Creating useful, stable and testable release gating jobs has proven to be difficult at scale. The combinatorial
-explosion of cloud providers, layered products and unique configurations means most jobs have a _>1% flake rate_. To "
-meet the tests where they are" the Technical Release Team has created a new type of job called the [_aggregated
-job_](/docs/release-oversight/improving-ci-signal#job-aggregation-in-a-little-more-detail).
+When considering whether payload blocking or component readiness is right for you, the
+primary concerns are detection speed and cost. 
 
-Aggregated Jobs are powerful because they allow a suite of tests to become blocking in spite of a high flake rate. This
-is accomplished by analyzing the output from numerous parallel runs to see if payload quality remains as good or better
-than the historical mean. If not, the next course of action is to isolate the changes and revert them. This allows other
-non-regressing changes to make their way into a release. More importantly it avoids the exponentially difficult task of
-debugging multiple regressions in parallel (eg, it's hard to debug a layered product regression on a platform that no
-longer installs properly).
+![job-pyramid.png](/job-pyramid.png)
+<small style="font-size: 0.5em;">
+<a href="https://www.vecteezy.com/free-vector/pyramid-chart">Pyramid Chart Vectors by Vecteezy</a>
+</small>
 
-### Identifying Which Jobs Should Not Gate a Release
+| Use Case            | Detection Speed | Cost | Mechanism                                |
+|---------------------|-----------------|------|------------------------------------------|
+| Must never regress  | Hours           | $$$$ | Release payload, aggregation             |
+| Must never break    | Hours           | $$$  | Release payload, once with retries       |
```

Comment:
not sure where or how you want to indicate it, but payload blocking allows regressions on a multi-week basis.  Component readiness does not.